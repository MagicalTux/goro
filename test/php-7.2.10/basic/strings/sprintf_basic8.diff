     *** Testing sprintf() : basic functionality - using octal format ***
     string(6) "format"
     string(2) "21"
004- string(14) "21 37777777431"
005- string(18) "21 37777777431 567"
004+ string(7) "21 -347"
005+ string(11) "21 -347 567"
     Done
