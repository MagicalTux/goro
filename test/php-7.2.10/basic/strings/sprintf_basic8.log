
---- EXPECTED OUTPUT
*** Testing sprintf() : basic functionality - using octal format ***
string(6) "format"
string(2) "21"
string(14) "21 37777777431"
string(18) "21 37777777431 567"
Done
---- ACTUAL OUTPUT
*** Testing sprintf() : basic functionality - using octal format ***
string(6) "format"
string(2) "21"
string(7) "21 -347"
string(11) "21 -347 567"
Done
---- FAILED
