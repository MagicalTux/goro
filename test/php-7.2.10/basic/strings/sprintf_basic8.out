*** Testing sprintf() : basic functionality - using octal format ***
string(6) "format"
string(2) "21"
string(7) "21 -347"
string(11) "21 -347 567"
Done